segmentation_and_shape/get_segmentations.ipynb

```diff
@@ -22,6 +23,10 @@
    "source": [
     "DATA_DIR= Path(r'\\\\allen\\aics\\assay-dev\\users\\Sandi\\pair-up\\camille\\comb_img/')\n",
     "WORKING_DIR = Path.cwd() / 'results'\n",
+    "# Change the paths if working on the virtual machine\n",
+    "if platform.node() == 'EC2AMAZ-SQ515AQ':\n",
```

Comment:
will this work across all virtual machines?

Reply:
Good question. I don't think so but we can use a list with names of other machines once we have them.

Reply:
If it helps, something like this is a pattern I've seen to test if you're on EC2 (since the list of machine names will probably be fairly dynamic):

```
from ec2_metadata import ec2_metadata

try:
    # This will raise an exception if not on an EC2 instance
    instance_id = ec2_metadata.instance_id
    print(f"This is an EC2 instance with ID: {instance_id}")
except:
    print("This is not an EC2 instance")
```

Reply:
This is great and more robust @anl. Will incorporate that.